dev-docs/create-gce-linux-vm.md

```diff
@@ -16,7 +16,7 @@ $ gcloud compute images list | grep $IMAGE_SEARCH_KEYWORD
 $ export OS_IMAGE_PROJECT=debian-cloud
 
 # VM image family.
-$ export OS_IMAGE_FAMILY=debian-10
+$ export OS_IMAGE_FAMILY=debian-12
```

Comment:
Should we not be testing on `debian-11` (the oldest available) by default? And then, why not `ubuntu-os-cloud:ubuntu-2004-lts` (which is actually older)?

Reply:
[e18feca](https://github.com/GoogleCloudPlatform/ops-agent/pull/1752/commits/e18feca17e427d43c5d2ed801276a535b2105733)